jvb/src/main/java/org/jitsi/videobridge/octo/ConfOctoTransport.java

```diff
@@ -557,13 +557,13 @@ OrderedJsonObject toJson()
             debugState.put("packets_received", packetsReceived);
             debugState.put("receive_packet_rate_pps", receivePacketRate.getRate());
             debugState.put("bytes_received", bytesReceived);
-            debugState.put("receive_bitrate_bps", receiveBitRate.getRate().getBps());
+            debugState.put("receive_bitrate_bps", receiveBitRate.getRate());
             debugState.put("incoming_packets_dropped", incomingPacketsDropped);
 
             debugState.put("packets_sent", packetsSent.sum());
             debugState.put("send_packet_rate_pps", sendPacketRate.getRate());
             debugState.put("bytes_sent", bytesSent.sum());
-            debugState.put("send_bitrate_bps", sendBitRate.getRate().getBps());
+            debugState.put("send_bitrate_bps", sendBitRate.getRate());
```

Comment:
These changes are taking advantage of the fact that the inlined `Bandwidth` looks like a `double` to Java - we're okay with that?

Reply:
I'm not sure I follow, what's the concern?

Reply:
This is code that is *dependent* on the fact that `Bandwidth` is an inline class; if it were changed to not be inlined, I believe either it wouldn't work or you'd end up with the non-numeric output of `Bandwidth#toString()` in the stats.

We could either add a `sendBitRate.getRateBps()` method, or decide we don't care.

Reply:
Ah gotcha.  It didn't occur to me that this was something we'd worry about, but I'm fine either way.

Reply:
I'm inclined to say let's not worry about it, as long as it's a conscious decision.